x-pack/plugins/global_search_bar/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "globalSearchBar",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
```

Comment:
You should update `.github/CODEOWNERS` to add `core-ui` as the owner of this plugin.